### PR TITLE
Xml avoid multicast 1.8.x

### PR DESCRIPTION
**Pull request by LuisGP**
Backport of #56

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/70/)

File: docs/xmlprofiles.rst [hunk 1]
```diff
@@ -718,6 +719,10 @@ This section of the :class:`Participant's rtps` configuration allows defining bu
 |                           | other discovered RTPSParticipants     |                         |                       |
 |                           | as well as to all Multicast ports.    |                         |                       |
 +---------------------------+---------------------------------------+-------------------------+-----------------------+
+| |avoidmc|                 | Ignores the multicast locators        | ``Boolean``             | :class:`true`         |
+|                           | received from the remote              |                         |                       |
+|                           | participants.                         |                         |                       |
++---------------------------+---------------------------------------+-------------------------+-----------------------+
```
Comment (raquelalvarezbanos) [inline docs/xmlprofiles.rst]: Check my comment in the other PR that goes against master

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/71/)

**Comment by LuisGP** [comment]
Tests fail in Jenkins, but in local passed.

**Pull request merged**